client/platform/desktop/constants.ts

```diff
@@ -0,0 +1,88 @@
+import { DatasetMeta } from 'viame-web-common/apispec';
+
+export const websafeVideoTypes = [
+  'video/mp4',
+  'video/webm',
+];
+
+export const websafeImageTypes = [
+  'image/apng',
+  'image/bmp',
+  'image/gif',
+  'image/jpeg',
+  'image/png',
+  'image/svg+xml',
```

Comment:
Are these really supported ?  Or are these a list of eventual things we want to support.  I don't believe the ImageAnnotator.js will support `svg`, or `bmp` types.  It should probably do gifs, but the others require a canvas and some additional logic we don't have right now.